cpp/box/Box.h

```diff
@@ -435,6 +445,31 @@ class Box
         );
     }
 
+    //! Get mask of points that fit inside the box.
+    /*! \param points Point positions.
+        \param n_points The number of points.
+        \param cropped_mask Mask of points inside the box.
+    */
+    void crop(const vec3<float>* points, const unsigned int n_points,
+        unsigned char* cropped_mask) const
+    {
+        util::forLoopWrapper(0, n_points, [&](size_t begin, size_t end) {
+            for (size_t i = 0; i < n_points; ++i)
```

Comment:
TBB alert. ⚠️ Need to be sure to use `begin` and `end` (not `0` to `n_points`) or else you'll have every parallel thread compute the entire range instead of just their chunk. 
```suggestion
            for (size_t i = begin; i < end; ++i)
```